src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java

```diff
@@ -56,4 +60,17 @@ public SendLetterController(
 
         return ok().body(messageId);
     }
+
+    @PutMapping(path = "/{id}/sent-to-print-at")
```

Comment:
Looks odd when not about the DB attribute. Is it normal to have `_at` suffix for the endpoint?

Reply:
especially when body also contains same thing - too many `sent-to-print-at`s 🙃

Reply:
I _could_ just have `value` or `date` in the Java model, but full field name in json model will be clearer from API spec point of view imho.

Reply:
body message all good as is. just wondering whether it's normal/ok to have such endpoint

Reply:
`sent-to-print` would look like a boolean to me at first glance.
Trying to make it as clear as possible.
Plus it matches the field that's getting updated.